compiler/rustc_codegen_ssa/src/mir/mod.rs

```diff
@@ -129,6 +129,7 @@ impl<'a, 'tcx, V: CodegenObject> LocalRef<'tcx, V> {
 
 ///////////////////////////////////////////////////////////////////////////
 
+#[tracing::instrument(level = "debug", skip(cx))]
```

Comment:
you can just use `instrument`, no need for the full path.

```suggestion
#[instrument(level = "debug", skip(cx))]
```